src/pages/reference/handlers/index.md

```diff
@@ -4,15 +4,16 @@ title: Handlers | API Mesh for Adobe Developer App Builder
 
 # Handlers
 
-Handlers, or source handlers, allow you to define sources that provide data to your mesh. The following table specifies the handlers supported by the API Mesh and the version of each handler:
+Handlers, or source handlers, allow you to define sources that provide data to your mesh. The following table specifies the handlers supported by API Mesh and the version of each handler:
 
 | Handler Package Name | Version |
 |------------|------------|
 [openapi] | `0.33.26`
 [graphql] | `0.31.24`
 [json-schema] | `0.35.38`
+[soap] | `0.14.25`
 
-[Source Handlers] provides more information about handlers in the context of the API Mesh.
+[Source Handlers] provides more information about handlers in the context of API Mesh.
```

Comment:
```suggestion
[Source handlers] provides more information about handlers in the context of API Mesh.
```

Changed this because you've been lowercasing headings/links throughout this PR. You would also need to update the link definition at the bottom of the file and also check for other links..